Intersect (Core)/GameObjects/ItemRange/ItemRange.cs

```diff
@@ -0,0 +1,38 @@
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Intersect.GameObjects.ItemRange;
```

Comment:
Please rename the folder (and namespace) that ItemRange and StatRange are in to `Ranges` so I never have to see code like `ItemRange.ItemRange` again (I already have to see `Interface.Interface`, possibly due to my own actions I can't remember, but I hate seeing it).